IntercomDotnet/Client.cs

```diff
@@ -15,9 +15,9 @@ public class Client
         public string UserName { get; set; }
         public string Password { get; set; }
 
-        public double GetUnixTimestamp(DateTime inputTime)
+        public long GetUnixTimestamp(DateTime inputTime)
         {
-            return (inputTime - new DateTime(1970, 1, 1).ToUniversalTime()).TotalSeconds;
+            return (long)inputTime.ToUniversalTime().Subtract(new DateTime(1970, 1, 1, 0, 0, 0)).TotalSeconds;
```

Comment:
When I have time I will implement a test library for this, and this would be perfect for an integration test - unfortunately I can't really automate these as it would require real access to Intercom.

Actually, I see you updated the ConsoleTest program, did this behave as expected with the above implementation?

If so, can you also try with no `ToUniversalTime()` calls, and also confirm you are using a non-UTC time zone?

Thanks for the PR!